can_testbench.py

```diff
@@ -146,21 +155,28 @@ def stop(self, msg):
         sendDetails = self.periodicMsgs.get(msg.arbitration_id)
         if sendDetails is not None:
             sendDetails['task'].stop()
+            
+    def shutdown(self):
+        self.notifier.stop()
+        self.bus.shutdown()
 
 
-class MsgModel(QAbstractTableModel):
+class MsgModel(QtCore.QAbstractTableModel):
     """
     A class that handles the data in a message table.  Can either be a message that
     is transmitted from the app or received by the app.
 
     Attributes:
-    signalValueChanged (Signal): A class attribute that represents the signal
-    to be sent if something in the table changes.
+    signalValueChanged (Qt.Signal): A class attribute that represents the signal
+    to be sent if the value of a can Signal in the table changes.
+    signalGraphedChanged (Qt.Signal): A signal to be sent if the graphed status
+    of a can Signal in the table changes.
     Columns (dict): A class attribute describing the columns in the table
     msg (DbcMessage): The message the table is displaying
     rxTable (bool): True if this is a table that describes messages the app receives
     """
-    SignalValueChanged = Signal(DbcMessage, int, object, object)
+    signalValueChanged = QtCore.Signal(DbcMessage, int, float)
+    signalGraphedChanged = QtCore.Signal(DbcMessage, int, bool, object)
```

Comment:
I split this into two signals because the original slot felt like it wanted to be two different functions for two different situations. This signal was only emitted twice, once with the first object as null and once with the second object as null.